SpatialGDK/Source/Private/Interop/SpatialReceiver.cpp

```diff
@@ -173,14 +175,12 @@ void USpatialReceiver::CreateActor(Worker_EntityId EntityId)
 	check(Position && Metadata);
 
 	AActor* EntityActor = EntityRegistry->GetActorFromEntityId(EntityId);
-	UE_LOG(LogTemp, Log, TEXT("!!! Checked out entity with entity ID %lld"), EntityId);
 
 	if (EntityActor)
 	{
 		UClass* ActorClass = GetNativeEntityClass(Metadata);
 
-		// Option 1
-		UE_LOG(LogTemp, Log, TEXT("Entity for core actor %s has been checked out on the worker which spawned it."), *EntityActor->GetName());
+		UE_LOG(LogSpatialReceiver, Log, TEXT("Entity for core actor %s has been checked out on the worker which spawned it."), *EntityActor->GetName());
```

Comment:
We should document what a core actor means so that this comment makes sense

Reply:
We should just remove the term. Potentially replace with `Unreal` in this instance.